### PR TITLE
GoldenRatio: Use structured_answer

### PR DESCRIPTION
**Pull request by marianosimone**
This addresses part of #1163 (Convert HTML Goodies to use templates)

https://duck.co/ia/view/golden_ratio

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@andrearonsen I believe this is your IA? Any objections to the changes?

File: lib/DDG/Goodie/GoldenRatio.pm [hunk 1]
```diff
@@ -4,26 +4,40 @@ package DDG::Goodie::GoldenRatio;
...
     my $result = 0;
 
     if ($input =~ /^(?:(?:(\?)\s*:\s*(\d+(?:\.\d+)?))|(?:(\d+(?:\.\d+)?)\s*:\s*(\?)))$/) {
-        if ($1 && $1 eq "?") {
+        if ($1 && $1 eq '?') {
             # ? : x
             $result = $2 / $golden_ratio;
-            return "Golden ratio: $result : $2";
-        } elsif ($4 && $4 eq "?") {
+            return answer("Golden ratio: $result:$2", "$result:$2", "Golden ratio for ?:$2");
```
Comment (mintsoft) [inline lib/DDG/Goodie/GoldenRatio.pm]: I'd probably want to pull the "Golden ratio for" string into the `answer` sub; but it's not a big deal

Comment (marianosimone) [inline lib/DDG/Goodie/GoldenRatio.pm]: Good idea!

**Comment by marianosimone** [comment]
who is in charge of merging PRs? @mintsoft ?

**Comment by gautamkrishnar** [comment]
Since this is assigned to @mintsoft, he will merge it.

**Pull request merged**